### ISSUE TITLE
Implement Collatz Backoff

### ISSUE THREAD

**Issue by alixaxel** [stanislavromanov/backoff#1]
Collatz would be awesome. xD

**Comment by stanislavromanov** [comment]
- Take a number
- If you can divide it by 2 than take it
- Else multiply by 3 and add one and then take it

Is this correct assumption?

**Comment by alixaxel** [comment]
Yes, it grows faster than it shrinks but it always ends up in 1 for some reason.

**Comment by stanislavromanov** [comment]
Well not really because if it's 1 then it will be 4 next iteration.

**Comment by alixaxel** [comment]
When you reach 1 you stop.
